README.md

```diff
@@ -5,23 +5,291 @@
+Istio using the SDS configuration profile using Helm:
+
+```bash
+helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+    --values install/kubernetes/helm/istio/values-istio-sds-auth.yaml | kubectl apply -f -
+```
+
+In the new API, the same profile would be selected through a CustomResource (CR):
+
+```yaml
+# sds-install-cr.yaml
+
+apiVersion: install.istio.io/v1alpha1
+kind: IstioControlPlane
+metadata:
+  name: istio-operator-config
+  namespace: istio-system
+spec:
+  profile: sds
+```
+
+CRs are used when running the operator as a controller in a pod in the cluster. When using operator CLI mode and passing the 
+configuration as a file (see [Select a profile](#Select_a_profile)), only the spec portion is required.
+
+```yaml
+# sds-install.yaml
+
+profile: sds
+```
+
+If you don't specify a configuration profile, Istio is installed using the `default` configuration profile. All 
+profiles listed in istio.io are available compiled in, or `profile:` can point to a local file path to reference a custom
+profile base to use as a starting point for customization. See the [API reference](https://github.com/istio/operator/blob/master/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto)
+for details.
+
+## Developer quick start
+
+The quick start describes how to install and use the operator `iop` CLI command. 
+
+#### Installation 
+
+```bash
+git clone https://istio.io/operator.git
+cd operator
+go build -o <your bin path> ./cmd/iop.go
+```
+
+#### Flags
+
+The `iop` command supports the following flags:
+
++   `logtostderr`: log to console (by default logs go to ./iop.go).
++   `dry-run`: console output only, nothing applied to cluster or written to files (default is true for now).
++   `verbose`: display entire manifest contents and other debug info (default is false).
+
+#### Basic default manifest
+
+The following command generates a manifest with the compiled in default profile and charts:
+
+```bash
+iop manifest
+```
+
+You can see these sources for the compiled in profiles in the repo under data/profiles, while the compiled in helm
+charts are under data/charts.
+
+#### Output to dirs
+
+The output of the manifest is concatenated into a single file. To generate a directory hierarchy with subdirectory
+levels representing a child dependency, use the following command:
+
+```bash
+iop manifest -o istio_manifests
+```
+
+Use depth first search to traverse the created directory hierarchy when applying your YAML files. Child manifest
+directories must wait for the parent, but not sibling manifest directories.
+
+#### Just apply it for me
+
+The following command generates the manifests and applies them in the correct dependency order, waiting for the
+dependencies to have the needed CRDs available: 
+
+```bash
+iop install
+```
+
+Note: right now the configuration that would be applied is only displayed, since `dry-run` is true by default. Set
```

Comment:
delete right now and capitalize `T`he.

Reply:
reorder the sentence such that the `dry-run=true` is started up front.  E.g. : `The default dry-run=true will force the display of the manifest only.  Set dry-run=false to apply the manifest to Istio.`

Reply:
```suggestion
The default `dry-run=true` displays the manifest only. Set `dry-run=false` to apply the Istio configuration to Kubernetes.
```